frontend/src/components/studySamples/StudyStepSamplesTable.tsx

```diff
@@ -99,8 +99,20 @@ function StudyStepSamplesTable({ studyID, step, settings }: StudyStepSamplesTabl
 			// is already in the project details page.
 			return [
 				...getColumnsForStudySamplesStep(stepDefinition, protocol),
-				actionColumn,
-			]
+				{
+					columnID: 'SAMPLE_COUNT',
+					title: 'Sample Count (in Pool)',
+					dataIndex: ['sample', 'id'],
+					render: (_, { sample }: SampleAndLibrary) => {
+						return (
+								sample && sample.is_pool
+								? sample.derived_samples_count
+								: '-'
```

Comment:
I'd prefer it if we just render an empty string, rather than "-" to be consistent with the rest of the UX.